lib/src/view/watch/tv_screen.dart

```diff
@@ -111,6 +113,21 @@ class _Body extends ConsumerWidget {
                       : null,
                   materialDiff: game.lastMaterialDiffAt(Side.white),
                 );
+
+                final gameAtCursor = game.positionAt(gameState.stepCursor);
+
+                final gameDataAtCursor = GameData(
```

Comment:
The `GameData` was meant only for boards that are playable. 

For now we can only display the `lastMove` which is already a great improvement. We'll only be missing the king check highlight which is not a big deal. I'll figure out what we can do with the chessground API later, to improve that.

Reply:
updated